client/utils/icons.ts

```diff
@@ -0,0 +1,67 @@
+import { TranslationKey } from '@/lang/flattenMessages'
+
+import { MeasurementsNoteEnum } from '@/enums/MeasurementNote.enum'
+
+export const icons = {
+  getNotesIconName: (note: MeasurementsNoteEnum) => {
+    switch (note) {
+      case MeasurementsNoteEnum.ALCOHOL:
+        return 'glass'
+      case MeasurementsNoteEnum.SICKNESS:
+        return 'vaccine'
+      case MeasurementsNoteEnum.SLEEP_DEPRIVATION:
+        return 'zzz'
+      case MeasurementsNoteEnum.STRESS:
+        return 'activity-heartbeat'
+      case MeasurementsNoteEnum.WATER_DEPLETION:
+        return 'droplet-half-2-filled'
+      case MeasurementsNoteEnum.MENSTRUATION:
+        return 'gender-female'
+      case MeasurementsNoteEnum.TRAVEL:
+        return 'plane-tilt'
+      default:
+        return ''
+    }
+  },
```

Comment:
What do you think about creating `lookup table` instead of `switch/case`?
https://blog.logrocket.com/evaluating-alternatives-typescript-switch-case/